javafx8-mapview/src/main/java/slash/navigation/mapview/browser/JavaFX8WebViewMapView.java

```diff
@@ -83,4 +85,8 @@ public void run() {
             }
         });
     }
+
+    protected void setBrowserScale(WebView webView, double scale) {
+        webView.setZoom(scale);
+    }
```

Comment:
die Methode gibt es in Java 7 nicht, weshalb man mit Java 7 nicht mehr kompilieren könnte